index.html

```diff
@@ -2331,7 +2331,14 @@ <h3>
 The <a>DID method</a> specification MUST specify how a client can update a
 <a>DID document</a> on the <a>DID registry</a>, including all cryptographic
 operations necessary to establish proof of control, <em>or</em> state that
-updates are not possible.
+updates are not possible. 
+        </p>
+
+        <p>
+An update to a DID is any change in the data used to produce a DID Document. 
```

Comment:
This sentence feels weird to me because it could be understood that the initial creation of a DID Document is an update.

Nothing => <basic did document>

Is that an intentional choice of language?

Reply:
I think the language aligns well with KERI / did:peer / did:ethr  / sidetree, but I'm happy to change it if there is something stronger.

Reply:
I added "after creation" to address this concern.

Reply:
Can I resolve this thread?